### ISSUE TITLE
Cannot call color() on a PGraphics

### ISSUE THREAD

**Issue by colouredmirrorball** [processing/processing#3762]
This code:

```
PGraphics pg = createGraphics(100,100);
pg.color(50);
```

results in the error "unexpected token: ." upon compiling. 

This code:

```
PGraphics pg = createGraphics(100,100);
int c = pg.color(50);
println(c);
```

results in the error "unexpected token: int".

**Comment by benfry** [comment]
@Manindra29 looks like your preproc is treating `color` as a type even though we have the function/cast version. Is that possible?

**Comment by Manindra29** [comment]
Yes, this is a known limitation of the parser. It's not good at identifying the function/cast usage of color. Taking a look.

**Comment by Manindra29** [comment]
@benfry I looked into this today. It seems the error is being thrown by the original preprocessor, not PDE X's preprocessor. No syntax errors are reported in the error table. The java source generated by ErrorCheckerService's preprocessor is:

````
PGraphics pg = createGraphics(100,100);
int c = pg.color(50);
````
